Vagrantfile

```diff
@@ -0,0 +1,78 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "fedora/34-cloud-base"
```

Comment:
This is fedora34 and not 32 as mentioned in the PR description and your commit message

Reply:
I fixed it

Reply:
Could you please change it in the commit message also? You can use [this guide](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) for rewriting git history?

Reply:
I did it..
I hope what I did was right